ramalama/cli.py

```diff
@@ -189,6 +190,14 @@ def configure_arguments(parser):
         action="store_true",
         help="offload the workload to the GPU",
     )
+
+    parser.add_argument(
```

Comment:
This should be part of ramalama info, no option equired, unless this takes a lot of time to call?

Reply:
still want to change the code @rhatdan ?

Reply:
Yes just add this to ramalama info.

Reply:
Tested with Intel:
```
   "GPUs": {
        "Detected GPUs": [
            {
                "Details": "00:02.0 VGA compatible controller: Intel Corporation TigerLake-H GT1 [UHD Graphics] (rev 01)",
                "Env": "ONEAPI_DEVICE_SELECTOR",
                "GPU": "Intel GPU",
                "VRAM": "Unknown"
            }
        ],
        "Errors": [
            {
                "Error": "Unable to detect NVIDIA GPU(s). Error: Unknown error (check if NVIDIA drivers are loaded).",
                "Vendor": "NVIDIA"
            },
            {
                "Error": "No AMD GPU detected or drivers missing.",
                "Vendor": "AMD"
            }
        ]
    },
```

Reply:
macos:

```
   "GPUs": {
        "Detected GPUs": [
            {
                "Cores": "32",
                "GPU": "Apple M1 Max",
                "Metal": "Metal 3",
                "Vendor": "Apple (0x106b)"
            },
            {
                "GPU": "Color LCD"
            }
        ],
        "Errors": "No errors"
    },
    "Image": "quay.io/ramalama/ramalama",
    "Runtime": "llama.cpp",
    "Store": "/var/lib/ramalama",
    "UseContainer": true,
    "Version": "0.5.4"
}

```

Reply:
@rhatdan I think we addressed this. AFAIK, macOS with Apple M(family) cannot have Intel or AMD together. However, it's possible to have hardware with AMD and Intel (not in the mainboard) as extra card. 
Please let me know your thoughts. I thought this change would be super easy but in the end, not :)